setup.py

```diff
@@ -60,7 +45,8 @@ def load_requirements(*requirements_paths):
         'Operating System :: OS Independent',
         'Programming Language :: Python',
     ],
-    packages=PACKAGES,
+    packages=find_packages(exclude=['*.test', '*.tests']),
```

Comment:
Just verified, and without any `exclude`, we get test packages included, and `blahblah/test/__init__.py` files are present.